esp-hal/src/uart.rs

```diff
@@ -954,6 +925,22 @@ where
     }
 }
 
+/// List of exposed UART events.
+#[derive(EnumSetType)]
```

Comment:
You don't wanna derive anything else?

Reply:
I just copied SpiInterrupt without thinking, may add derives there as well.

Reply:
Good guy EnumSetType gives us a lot for a single derive, I always forget that.